openchemistry/__init__.py

```diff
@@ -668,6 +668,32 @@ def _find_using_cactus(identifier):
     else:
         return None
 
+def import_structure(inchikey=None, inchi=None, smiles=None):
+
+    params = {}
+    if inchikey:
+        params['inchikey'] = inchikey
+    if inchi:
+        params['inchi'] = inchi
+    if smiles:
+        params['smiles'] = smiles
+
+    if not params:
+        raise Exception('At least one of these must be set: '
+                        'inchikey, inchi, or smiles')
+
+    molecules = girder_client.get('molecules', parameters=params)
```

Comment:
Can we use the POST ( create endpoint? ) here?

Reply:
Oh yes... I have that in my notes. Sorry I missed that.

Reply:
It is fixed. Now, if the molecule doesn't exist in the database, one will be created.